openreview/venue_request/process/revisionProcess.py

```diff
@@ -42,6 +42,11 @@ def process(client, note, invitation):
 
         elif invitation_type == 'Decision_Stage':
             conference.set_decision_stage(openreview.helpers.get_decision_stage(client, forum_note))
+            decisions_file = forum_note.content.get('upload_decisions')
```

Comment:
move this behavior to the `DecisionStage` class so we don't need to update the revision process function and the functionality keeps encapsulated.